aries-site/src/data/wcag/toolbar.json

```diff
@@ -0,0 +1,512 @@
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.7",
+    "label": "Dragging Movements",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.8",
+    "label": "Target Size (Minimum)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.1",
+    "label": "Language of Page",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.2",
+    "label": "Language of Parts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.3",
+    "label": "Unusual Words",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.4",
+    "label": "Abbreviations",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.5",
+    "label": "Reading Level",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.6",
+    "label": "Pronunciation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.1",
+    "label": "On Focus",
+    "status": "not-applicable",
+    "notes": "toolbar with not get focus"
+  },
+  {
+    "rule": "3.2.2",
+    "label": "On Input",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.3",
+    "label": "Consistent Navigation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.4",
+    "label": "Consistent Identification",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.5",
+    "label": "Change on Request",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.6",
+    "label": "Consistent Help",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.1",
+    "label": "Error Identification",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.2",
+    "label": "Labels or Instructions",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.3",
+    "label": "Error Suggestion",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.4",
+    "label": "Error Prevention (Legal, Financial, Data)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.5",
+    "label": "Help",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.6",
+    "label": "Error Prevention (All)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.7",
+    "label": "Redundant Entry",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.8",
+    "label": "Accessible Authentication (Minimum)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.9",
+    "label": "Accessible Authentication (Enhanced)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "4.1.2",
+    "label": "Name, Role, Value",
+    "status": "failed",
+    "notes": "we should add the role toolbar "
```

Comment:
Might be good to discuss this one a little more...

I was looking at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/toolbar_role and noticed that it says the following: 
> "Only use the toolbar role to group 3 or more controls"

We have more than 3 controls in our toolbar

> "can help reduce the number of tab stops for keyboard users"

Does this mean that if something has the toolbar role we should navigate via arrow keys within. the toolbar

> "The toolbar is commonly a subset of functions found in a menubar"

We aren't using menubar

Reply:
ah yes this would be good to talk through we can chat tomorrow

Reply:
I read a little more and agree that if we for example only have a filter in a toolbar this may actually confuse screen reader users, who are told there's a toolbar (implying multiple tools), but only find one control.

from what I captured you can still use `toolbar` even without having the menubar.

> can help reduce the number of tab stops for keyboard users

>Does this mean that if something has the toolbar role we should navigate via arrow keys within. the toolbar
So I added the `toolbar` role to this story locally https://storybook.grommet.io/?path=/story/data-toolbar-simple--simple&globals=theme:hpe
and it does let you know that you are in a `toolbar` and lets you know that you can use arrow keys to go through options in `toolbar` but I was still able to use `tab` as well

Reply:
@jcfilben changed to conditional and added comment